mlir/lib/Dialect/Rock/Transforms/AffixTuningParameters.cpp

```diff
@@ -135,8 +135,10 @@ void AffixTuningParameters::affixTuningParametersImpl(
         status = populateParamsAccelPtr->obtainTuningParameters(
             op, perfConfig, validParams, blockSize);
       }
-      if (failed(status))
+      if (failed(status)) {
         signalPassFailure();
+        return;
```

Comment:
The particular combination of cmake flags that I've been using has sent me down this path for a couple of tests.  I'm apparently doing combinations that CI doesn't usually do.

Most uses of signalPassFailure() return immediately, but these don't, leading to crashes later on.  With the returns, they just report errors, which I'll have to do something about, but at least it's cleaner.